src/Tizen.CoreUI/src/CoreUI.UI/AlertPopup.cs

```diff
@@ -0,0 +1,405 @@
+[SuppressMessage("Microsoft.Naming", "CA1724:TypeNamesShouldNotMatchNamespaces")]
+public class AlertPopup : CoreUI.UI.Popup
+{
+    /// <summary>Pointer to the native class description.</summary>
+    public override System.IntPtr NativeClass
+    {
+        get
+        {
+            if (((object)this).GetType() == typeof(AlertPopup))
+            {
+                return GetCoreUIClassStatic();
+            }
+            else
+            {
+                return CoreUI.Wrapper.ClassRegister.klassFromType[((object)this).GetType()];
+            }
+        }
+    }
+
+    [System.Runtime.InteropServices.DllImport(CoreUI.Libs.Elementary)] internal static extern System.IntPtr
+        efl_ui_alert_popup_class_get();
+
+    /// <summary>Initializes a new instance of the <see cref="AlertPopup"/> class.
+    /// </summary>
+    /// <param name="parent">Parent instance.</param>
+    /// <since_tizen> 6 </since_tizen>
+/// <param name="style">The widget style to use. See <see cref="CoreUI.UI.Widget.SetStyle" /></param>
+    public AlertPopup(CoreUI.Object parent, System.String style = null) : base(efl_ui_alert_popup_class_get(), parent)
+    {
+        if (CoreUI.Wrapper.Globals.ParamHelperCheck(style))
+        {
+            SetStyle(CoreUI.Wrapper.Globals.GetParamHelper(style));
+        }
+
+        FinishInstantiation();
+    }
+
+    /// <summary>Subclasses should override this constructor if they are expected to be instantiated from native code.
+    /// Do not call this constructor directly.
+    /// </summary>
+    /// <param name="ch">Tag struct storing the native handle of the object being constructed.</param>
+    /// <since_tizen> 6 </since_tizen>
+    protected AlertPopup(ConstructingHandle ch) : base(ch)
+    {
+    }
+
+    /// <summary>Initializes a new instance of the <see cref="AlertPopup"/> class.
+    /// Internal usage: Constructs an instance from a native pointer. This is used when interacting with C code and should not be used directly.
+    /// </summary>
+    /// <param name="wh">The native pointer to be wrapped.</param>
+    internal AlertPopup(CoreUI.Wrapper.WrappingHandle wh) : base(wh)
+    {
+    }
+
+    /// <summary>Initializes a new instance of the <see cref="AlertPopup"/> class.
+    /// Internal usage: Constructor to forward the wrapper initialization to the root class that interfaces with native code. Should not be used directly.
+    /// </summary>
+    /// <param name="baseKlass">The pointer to the base native Eo class.</param>
+    /// <param name="parent">The CoreUI.Object parent of this instance.</param>
+    protected AlertPopup(IntPtr baseKlass, CoreUI.Object parent) : base(baseKlass, parent)
+    {
+    }
+
+    /// <summary>Called when an Alert_Popup button was clicked.</summary>
+    /// <since_tizen> 6 </since_tizen>
+    /// <value><see cref="CoreUI.UI.AlertPopupButtonClickedEventArgs"/></value>
+    public event EventHandler<CoreUI.UI.AlertPopupButtonClickedEventArgs> ButtonClickedEvent
```

Comment:
```suggestion
    public event EventHandler<CoreUI.UI.AlertPopupButtonClickedEventArgs> ButtonClicked
```
Please remove `Evnet` suffix